db/migrate/20240530151252_add_notes_input_type_to_activities.rb

```diff
@@ -0,0 +1,5 @@
+class AddNotesInputTypeToActivities < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
If rolled back, comments will still be filled with string containing special syntaxes, I suppose? Perhaps these should be processed or replaced. I don't know what the norm is.

Reply:
The special syntax is , or \n seperated. Don't think we should do any special processing to ensure backwards compatibility